app/views/step_by_step_pages/edit.html.erb

```diff
@@ -9,7 +9,7 @@
       href: @step_by_step_page
     },
     {
-      text: 'Edit nav'
+      text: 'Edit title, intro or slug'
```

Comment:
Is this change related to the navigation changes?

Reply:
hey, yes as part of the tabbing change - the rename was required.

Reply:
Why?